lib/suspenders/generators/app_generator.rb

```diff
@@ -67,11 +72,15 @@ def customize_gemfile
     def setup_database
       say 'Setting up database'
 
-      if 'postgresql' == options[:database]
-        build :use_postgres_config_template
-      end
+      if 'mongodb' == options[:database]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.